python/examples/pylammps/elastic/elastic.py

```diff
@@ -0,0 +1,309 @@
+    parser.add_argument("kim_model", help="the KIM ID of the interatomic model archived in OpenKIM")
+    parser.add_argument("elements", nargs='+', default=['Au'], help="a list of N chemical species, which defines a mapping between atom types in LAMMPS to the available species in the OpenKIM model")
+    parser.add_argument("--min_style", default="cg", help="which algorithm will be used for minimization from lammps")
+    parser.add_argument("--minimize", type=float, nargs=4, default=[1.0e-4, 1.0e-6, 100, 1000], help="minimization parameters")
+    parser.add_argument("--up", type=float, default=1.0e-6, help="the deformation magnitude (in strain units)")
+    args = parser.parse_args()
+
+    L = PyLammps()
+
+    L.units("metal")
+
+    # Define the finite deformation size.
+    #Try several values to verify that results do not depend on it.
+    L.variable("up equal {}".format(args.up))
+
+    # Define the amount of random jiggle for atoms. It prevents atoms from staying on saddle points
+    atomjiggle = 1.0e-5
+
+    # metal units, elastic constants in GPa
+    cfac = 1.0e-4
+
+    # Define minimization parameters
+    L.variable("dmax equal 1.0e-2")
+
+    L.boundary("p", "p", "p") # periodic boundary conditions in all three directions
+    L.box("tilt large") # to avoid termination if the final simulation box has a high tilt factor
+
+    # use the OpenKIM model to set the energy interactions
+    L.kim_init(args.kim_model, "metal", "unit_conversion_mode")
+
+    L.read_data(args.input_data_file)
+
+    potential(L, args)
+
+    # Need to set mass to something, just to satisfy LAMMPS
+    mass_dictionary = {'H': 1.00797, 'He': 4.00260, 'Li': 6.941, 'Be': 9.01218, 'B': 10.81, 'C': 12.011, 'N': 14.0067, 'O': 15.9994, 'F': 18.998403, 'Ne': 20.179, 'Na': 22.98977, 'Mg': 24.305, 'Al': 26.98154, 'Si': 28.0855, 'P': 30.97376, 'S': 32.06, 'Cl': 35.453, 'K': 39.0983, 'Ar': 39.948, 'Ca': 40.08, 'Sc': 44.9559, 'Ti': 47.90, 'V': 50.9415, 'Cr': 51.996, 'Mn': 54.9380, 'Fe': 55.847, 'Ni': 58.70, 'Co': 58.9332, 'Cu': 63.546, 'Zn': 65.38, 'Ga': 69.72, 'Ge': 72.59, 'As': 74.9216, 'Se': 78.96, 'Br': 79.904, 'Kr': 83.80, 'Rb': 85.4678, 'Sr': 87.62, 'Y': 88.9059, 'Zr': 91.22, 'Nb': 92.9064, 'Mo': 95.94, 'Tc': (98), 'Ru': 101.07, 'Rh': 102.9055, 'Pd': 106.4, 'Ag': 107.868, 'Cd': 112.41, 'In': 114.82, 'Sn': 118.69, 'Sb': 121.75, 'I': 126.9045, 'Te': 127.60, 'Xe': 131.30, 'Cs': 132.9054, 'Ba': 137.33, 'La': 138.9055, 'Ce': 140.12, 'Pr': 140.9077, 'Nd': 144.24, 'Pm': (145), 'Sm': 150.4, 'Eu': 151.96, 'Gd': 157.25, 'Tb': 158.9254, 'Dy': 162.50, 'Ho': 164.9304, 'Er': 167.26, 'Tm': 168.9342, 'Yb': 173.04, 'Lu': 174.967, 'Hf': 178.49, 'Ta': 180.9479, 'W': 183.85, 'Re': 186.207, 'Os': 190.2, 'Ir': 192.22, 'Pt': 195.09, 'Au': 196.9665, 'Hg': 200.59, 'Tl': 204.37, 'Pb': 207.2, 'Bi': 208.9804, 'Po': (209), 'At': (210), 'Rn': (222), 'Fr': (223), 'Ra': 226.0254, 'Ac': 227.0278, 'Pa': 231.0359, 'Th': 232.0381, 'Np': 237.0482, 'U': 238.029}
```

Comment:
any particular reason why some values are in parentheses? E.g. `'Pm': (145)`

Reply:
If I am not mistaken (it has been 30 years since I last dealt with this), that is just a Chemistry/Physics convention. The masses in the periodic table usually are the mass due to the natural mix of isotopes for a given element. Some elements are rather short lived so that there is no "natural mix" (or no natural occurrence at all). Then the convention is to provide the mass of the most long lived isotope in parenthesis.

Reply:
Actually I should remove the parentheses. In the unlikely event someone finds a universal FF in OpenKIM the parentheses will lead to an error

Reply:
This is quite interesting... Since we're going back, I do remember that nuclear binding forces will measurably offset the total mass of the nucleus based on how much it is stable. E = mc^2, and although most of it is mass of protons and neutrons, the contribution due to nuclear forces is not negligible. The latter results in a higher/lower total mass for the nucleus than you would expect. Further, protons and neutrons have unequal masses and when increase in mass their optimal mix deviates from the exact 1:1 ratio that you have in lighter elements.

So how come those masses in parenthesis happen to be all exact multiples of the Dalton? Some of those elements are long lived enough for more precise mass measurements, I would think.